Jenkinsfile

```diff
@@ -298,6 +298,11 @@ pipeline {
                                 runTest("ethercat", 1, true)
                             }
                         }
+                        stage('EtherCAT Multislave') {
+                            steps {
+                                runTest("multislave", 0, true)
```

Comment:
How can you differentiate ecat multislave and canopen multislave? Now we only have ecat multislave, but in the future we will have canopen pdos, and maybe we will implement multislave tests.

Reply:
I get your point. Once we adopt the way ingeniamotion is currently setting up the drive setups, this will be solved. This is an easy fix until that issue is done.